src/components/Header/index.tsx

```diff
@@ -250,19 +230,17 @@ function Header({ history }: { history: any }) {
   const userEthBalance = useETHBalances(account ? [account] : [])?.[account ?? '']
   const [isDark] = useDarkModeManager()
 
-  const handleDisabledAnchorClick = useCallback(event => {
-    event.preventDefault()
-  }, [])
+  // const handleDisabledAnchorClick = useCallback(event => {
+  //   event.preventDefault()
+  // }, [])
 
   return (
     <HeaderFrame>
       <HeaderRow isDark={isDark}>
         <Title href=".">
-          <Link id="link" to="/">
-            <DXswapIcon>
-              <img src={isDark ? LogoDark : Logo} alt="logo" />
-            </DXswapIcon>
-          </Link>
+          <DXswapIcon>
```

Comment:
Why was the link removed here?

Reply:
![image](https://user-images.githubusercontent.com/54032883/103965876-86a32780-5113-11eb-9b4b-34a555a8042f.png)
Error from original code with <Link>.
Actually without Link, it works as desired.